docs/ktlint-setup-and-using.md

```diff
@@ -0,0 +1,9 @@
+## [Ktlint](https://github.com/pinterest/ktlint) setup
+
+1. On Mac OS or Linux: _brew install ktlint_
+1. Inside Project's root directory: _ktlint --apply-to-idea-project --android_ 
```

Comment:
NIT Typo should be `2.`

Reply:
it's not NIT, it's [markdown](https://daringfireball.net/projects/markdown/syntax#list) 👓